### PR TITLE
IOSXE/show wireless stats mobility

### PR DESCRIPTION
**Pull request by robschn**
## Description
Adding parser for command `show wireless stats mobility`.

## Motivation and Context
Converts raw output of `show wireless stats mobility` into structured data.

## Impact (If any)
None

## Screenshots:
image

## Checklist:

- [x] I have updated the changelog.
- [x] I have updated the documentation (If applicable).
- [x] I have added tests to cover my changes (If applicable).
- [x] All new and existing tests passed.
- [x] All new code passed compilation.

### CODE REVIEW

File: src/genie/libs/parser/iosxe/tests/ShowWirelessStatsMobility/cli/equal/golden_output_output.txt [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    Roam stats
+        L2 roam count                 : 0               
+        L3 roam count                 : 0               
+        Flex client roam count        : 0               
+        Inter-WNCd roam count         : 0               
+        Intra-WNCd roam count         : 154227          
+        Remote inter-cntrl roam count : 0               
+        Remote WebAuth pending roams  : 0               
+    Anchor Request
+        Sent                          : 0
```
Comment (robschn) [inline src/genie/libs/parser/iosxe/tests/ShowWirelessStatsMobility/cli/equal/golden_output_output.txt:22]: Please let me know how we should handle these lines. They have numbers associated, but the indenting would suggest that the values below should be part of a `"sent"` and `"received"` dict

Comment (LenzDong) [reply src/genie/libs/parser/iosxe/tests/ShowWirelessStatsMobility/cli/equal/golden_output_output.txt:22]: Hi, is it a total count for ```Grant received``` and ```Deny received```? If yes, you can add a total key under sent dict
```
 {
    "anchor_request": {
        "sent": {
            "grant_received": 0,
            "deny_received": 0,
            "total": 0,
        }
        "received": {
            "grant_sent": 0,
            "deny_sent": 0,
            "total": 0,
        },
    }
}

Comment (robschn) [reply src/genie/libs/parser/iosxe/tests/ShowWirelessStatsMobility/cli/equal/golden_output_output.txt:22]: It looks like a separate count, not a total. Might be better to leave as is.
image
This is the only resource I could find on the topic: 
http://what-when-how.com/deploying-and-troubleshooting-cisco-wireless-lan-controllers/troubleshooting-mobility-cisco-wireless-lan-controllers-part-1/

Comment (LenzDong) [reply src/genie/libs/parser/iosxe/tests/ShowWirelessStatsMobility/cli/equal/golden_output_output.txt:22]: Ok, sounds good! Thank you for the information.

File: src/genie/libs/parser/iosxe/show_wireless.py [hunk 2]
```diff
@@ -2053,3 +2053,285 @@ def cli(self, output=None):
...
+            header_capture_list = [
+                mobility_event_statistics_capture,
+                mm_mobility_event_statistics_capture,
+                mmif_mobility_event_statistics_capture,
+            ]
+
+            for capture in header_capture_list:
+                if capture.match(line):
+                    line_format = line.replace(" ", "_").lower().strip(":")
+                    wireless_info_obj.update({line_format: {}})
```
Comment (LenzDong) [inline src/genie/libs/parser/iosxe/show_wireless.py:2277]: You can use ```setdefault``` to combine the following lines:
```
header_group = wireless_info_obj.setdefault(line_format, {})
header_tracking = "header"

File: src/genie/libs/parser/iosxe/show_wireless.py [hunk 3]
```diff
@@ -2053,3 +2053,285 @@ def cli(self, output=None):
...
+                handoff_status_received_capture,
+                handoff_status_sent_capture,
+                export_anchor_capture,
+                mmif_internal_errors_capture,
+            ]
+
+            for capture in subheader_capture_list:
+                if capture.match(line):
+                    line_format = line.replace(" ", "_").lower().strip(":")
+                    header_group.update({line_format: {}})
```
Comment (LenzDong) [inline src/genie/libs/parser/iosxe/show_wireless.py:2297]: same here to use ```setdefault```

**Review by LenzDong — APPROVED** [review]

File: src/genie/libs/parser/iosxe/show_wireless.py (hunk 2 above)
Comment (robschn) [reply src/genie/libs/parser/iosxe/show_wireless.py]: Very cool, I appreciate the tip! Updated!

File: src/genie/libs/parser/iosxe/show_wireless.py (hunk 3 above)
Comment (robschn) [reply src/genie/libs/parser/iosxe/show_wireless.py]: Updated!

**Review by tahigash — APPROVED** [review]
schema looks good. @LenzDong please do code-review.

**Pull request merged**